_schemaV2.graphql

```diff
@@ -3601,6 +3601,31 @@ type DaySchedule {
   dayOfWeek: String
 }
 
+# Autogenerated input type of DebitCommissionExemption
+input DebitCommissionExemptionInput {
```

Comment:
What are all these changes? Are these things added to Gravity that weren't stitched into MP, and now are? If so, we might leave them out.

Reply:
@mzikherman we decided to prevent these unused things from being stitched in, and did so by splitting `removeTypes` into:

1. `duplicateTypes`: a list of types that are filtered because they override Metaphysics, and
2. `unusedTypes`: a list of types (and inputs) that are filtered because they are not used by Metaphysics